gdml-generator/src/NotionalJUNO.cc

```diff
@@ -0,0 +1,104 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024-2025 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file NotionalJUNO.cc
+//---------------------------------------------------------------------------//
+#define _USE_MATH_DEFINES
+
+#include "NotionalJUNO.hh"
+
+#include <cmath>
+#include <G4LogicalVolume.hh>
+#include <G4NistManager.hh>
+#include <G4Orb.hh>
```

Comment:
Looks like `G4VisAttributes` is not used. (I may need to revisit other geometries to make sure I also did not leave useless headers around there too)